pkg/addons2/embeddedclusteroperator/metadata.go

```diff
@@ -0,0 +1,49 @@
+package embeddedclusteroperator
+
+import (
+	k0sv1beta1 "github.com/k0sproject/k0s/pkg/apis/k0s/v1beta1"
+	"github.com/pkg/errors"
+	ecv1beta1 "github.com/replicatedhq/embedded-cluster/kinds/apis/v1beta1"
+	"github.com/replicatedhq/embedded-cluster/pkg/helm"
+	"k8s.io/utils/ptr"
+)
+
+func Version() map[string]string {
+	return map[string]string{
+		"EmbeddedClusterOperator": "v" + Metadata.Version,
+	}
+}
+
+func GetImages() []string {
+	var images []string
+	for _, image := range Metadata.Images {
+		images = append(images, image.String())
+	}
+	return images
+}
+
+func GetAdditionalImages() []string {
+	var images []string
+	if image, ok := Metadata.Images["utils"]; ok {
+		images = append(images, image.String())
+	}
+	return images
+}
+
+func GenerateChartConfig() ([]ecv1beta1.Chart, []k0sv1beta1.Repository, error) {
+	values, err := helm.MarshalValues(helmValues)
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "marshal helm values")
+	}
+
+	chartConfig := ecv1beta1.Chart{
+		Name:         releaseName,
+		ChartName:    Metadata.Location,
+		Version:      Metadata.Version,
+		Values:       string(values),
+		TargetNS:     namespace,
+		ForceUpgrade: ptr.To(false),
```

Comment:
This is set to false while it is set to true in pkg/addons2/embeddedclusteroperator/upgrade.go on line 37

```
	_, err = hcli.Upgrade(ctx, helm.UpgradeOptions{
		ReleaseName:  releaseName,
		ChartPath:    e.ChartLocation(),
		ChartVersion: e.ChartVersion(),
		Values:       values,
		Namespace:    namespace,
		Labels:       getBackupLabels(),
		Force:        true,
	})
```

Reply:
oh interesting. we should probably change that to false.